prov/gni/src/gnix_ep.c

```diff
@@ -360,15 +359,12 @@ static void __gnix_vc_destroy_ht_entry(void *val)
  * EP vc initialization helper
  ******************************************************************************/
 
-static inline int __gnix_ep_init_vc(struct gnix_fid_ep *ep_priv)
+int _gnix_ep_init_vc(struct gnix_fid_ep *ep_priv, enum fi_av_type type)
```

Comment:
I **think** Howard made this change. I agree it's not needed here.
@hppritcha, was there a reason for this change? Otherwise I can change it back.